fiftyone/multimodal/odm/mongo.py

```
@@ -0,0 +1,61 @@
+"""
+| Copyright 2017-2026, Voxel51, Inc.
+| `voxel51.com <https://voxel51.com/>`_
+|
+"""
+
+from typing import List, Optional
```

Ruff (0.15.12)

[warning] `typing.List` is deprecated, use `list` instead

(UP035)